### ISSUE TITLE
scalapack: make test fails on Linuxbrew

### ISSUE THREAD

**Issue by sjackman** [Homebrew/homebrew-science#3425]
https://gist.github.com/sjackman/05b6aa554ac47b7b74f1
```
0% tests passed, 96 tests failed out of 96
```

**Comment by dpo** [comment]
Is it still the case with `openblas`?

**Comment by sjackman** [comment]
So far 92/96 have passed! It's looking good. The last four tests are in progress…

**Comment by sjackman** [comment]
```
      Start 92: xdsyevr
92/96 Test #92: xdsyevr ..........................   Passed  140.60 sec
      Start 93: xcheevr
93/96 Test #93: xcheevr ..........................***Timeout 1500.07 sec
      Start 94: xzheevr
```

**Comment by dpo** [comment]
Should be fixed in #4519.
